extensions/TheShovel/ShovelUtilsTW.js

```diff
@@ -0,0 +1,257 @@
+console.log("ShovelUtils v1.1")
+tempImageLoad = null
```

Comment:
```suggestion
var tempImageLoad = {};
```
fixes tempImageLoad is not defined
also it appears to be completely unused